### PR TITLE
fix-#379: Enhancing the footer by including GitHub and Discord for better feedback

### PR DESCRIPTION
**Pull request by Lanthoiba2022**
## Summary

Enhancing the footer by including GitHub and Discord for better feedback

## Description

Added GitHub clickable icon which will take the person to GitHub for raising issue if any and
Discord clickable icon to connect with the community if desire.

## Images

Before

image

After

image

## Issue(s) Addressed

#379 

- **Closes #379**

## Prerequisites

- [x] Have you followed all the [CONTRIBUTING GUIDELINES](https://github.com/krishnaacharyaa/wanderlust/blob/main/.github/CONTRIBUTING.md#guidelines-for-contributions)?

### CODE REVIEW

File: frontend/src/layouts/footer-layout.tsx [hunk 1]
```diff
@@ -3,12 +3,48 @@ function footer() {
...
+      <div className="mb-8 mt-8">
+        <section className="flex items-center text-xs sm:text-sm">
+          Find an issue with this page?{' '}
+          <span className="pl-1 text-blue-500">
+            <a href="https://github.com/krishnaacharyaa/wanderlust">Fix it on GitHub</a>
+          </span>
+          <span>
+            <a href="https://github.com/krishnaacharyaa/wanderlust">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
```
Comment (krishnaacharyaa) [inline frontend/src/layouts/footer-layout.tsx:16]: Please extract this out to a different svg file and then import like other svg files in the project

File: frontend/src/layouts/footer-layout.tsx [hunk 2]
```diff
@@ -3,12 +3,48 @@ function footer() {
...
+          </span>
+        </section>
+
+        <section className="flex items-center pl-8 text-xs sm:text-sm">
+          Need help? Connect with us on
+          <span>
+            <a href="https://discord.com/invite/FEKasAdCrG">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                className="h-6 w-10"
```
Comment (krishnaacharyaa) [inline frontend/src/layouts/footer-layout.tsx:33]: Same comment

File: frontend/src/layouts/footer-layout.tsx [hunk 3]
```diff
@@ -3,12 +3,48 @@ function footer() {
...
+              </svg>
+            </a>
+          </span>
+        </section>
+
+        <section className="mt-4 flex items-center pl-16 text-xs">
+          <span className="mr-2">&copy;</span>
+          {year} All Rights Reserved
+        </section>
```
Comment (krishnaacharyaa) [inline frontend/src/layouts/footer-layout.tsx:46]: Change it to 2024

**Review by krishnaacharyaa — CHANGES_REQUESTED** [review]
Thank you @Lanthoiba2022 for the PR, and for the patience to follow all the guidelines, much appreciated
Kindly address the review comments

File: frontend/src/layouts/footer-layout.tsx (hunk 1 above)
Comment (Lanthoiba2022) [reply frontend/src/layouts/footer-layout.tsx:16]: Yeah sure

File: frontend/src/layouts/footer-layout.tsx (hunk 3 above)
Comment (Lanthoiba2022) [reply frontend/src/layouts/footer-layout.tsx:46]: year is taken using this method "const year = newDate.getFullYear();". Ig it's not to be hardcoded.

Comment (krishnaacharyaa) [reply frontend/src/layouts/footer-layout.tsx:34]: Oh okay, sorry missed it

**Review by krishnaacharyaa — APPROVED** [review]
LGTM, thank you for the efforts :)

**Pull request merged**